### ISSUE TITLE
missing file for pi3 v5.4?

### ISSUE THREAD

**Issue by Bazil79** [BlitterStudio/amiberry#1036]
Hi im new to the Pi,
i have installed the bullseye OS and downloaded the amiberry v5.4 and unpacked following a vid from youtube but i cant seem to find the executive file
i have a green amiberry logo but when i click on it nothing happens, what am i missing.

Please help

![20221211_151756 1](https://user-images.githubusercontent.com/120318822/206912287-d2dfc302-7ff4-452d-93c3-27595e02da8e.jpg)

**Comment by solskogen** [comment]
Open a terminal and run it from there. The green icon is the correct one, so you might miss some dependencies. Running it in terminal would help figure that out.

**Comment by midwan** [comment]
@Bazil79 
No file is missing, that `amiberry` file you see there is the executable. Like @solskogen said, it might be trying to run but fails due to some error (e.g. you're missing a requirement), but you don't see that because it's output on the console. 
It might also be a case of changing that file permissions, to allow it to be Executable. The ZIP packaging might not include that property.

You can do that either from a console or the GUI:
From console: `chmod a+x ./amiberry` (while being in the same directory as that file is)
From the GUI: Right click on it, select Permissions, check that Executable is enabled.

**Comment by Bazil79** [comment]
Many thanks for the replies, have checked the permissions and seem ok.![20221212_111525.jpg](https://user-images.githubusercontent.com/120318822/207032740-eac4ff51-358c-492e-a73b-e0ee7504819b.jpg)
Have tried running command but not sure if I done it correctly ![20221212_111756.jpg](https://user-images.githubusercontent.com/120318822/207033019-6996df28-72a7-4f56-b424-60af888ef979.jpg)

Many thanks barry

**Comment by Bazil79** [comment]
Ok so I did a fresh install of bullseye 32bit full from pi program.
Downloaded amiberry 32bit for pie
Unzipped and copied across kickstart files from amiga forever.
Checked permissions - all ok
Tried to open - nothing 
Opened terminal in amiberry window and used command like advised
![20221213_160049.jpg](https://user-images.githubusercontent.com/120318822/207382754-50d2cd98-efe3-421a-8465-5543471ee510.jpg)
Still nothing, even tried run a command in amiberry still nothing. 

Am I still missing something? Or is it not compatible with my pi?

**Comment by midwan** [comment]
1) You will have to install the requirements, as mentioned in the Readme: https://github.com/BlitterStudio/amiberry/blob/master/docs/README.md

2) Open a terminal, "cd" in the directory you have Amiberry, then run it like so: `./amiberry`
That should bring up the main GUI, or if it fails, at least output some information in the console (e.g. if a requirement is missing)

3) You will of course also need some Kickstart ROMs, which should be copied in the `kickstarts` directory. After you've done that, run Amiberry and from the GUI go to Paths -> Rescan ROMs. You only need to do that once, it will cache any roms it finds for future reference.

**Comment by Bazil79** [comment]
Thanks again for the response, I am obviously too thick.
I read the read me as stated went to the SDL2 site got confused so googled which version to use 
![20221214_115619.jpg](https://user-images.githubusercontent.com/120318822/207589312-5e48a20b-5ec5-4aec-90ba-99ead859991e.jpg)
Selected what I thought to be correct and downloaded, didn't know where to extract to so extracted to home/myself 
![20221214_120000.jpg](https://user-images.githubusercontent.com/120318822/207590006-1157108c-8476-4c86-a61a-9d3211c1424f.jpg)
Ran terminal in amiberry and got error 
![20221214_120248.jpg](https://user-images.githubusercontent.com/120318822/207590522-08c0b88e-217f-4492-9cac-0dda3b2f5bc4.jpg)

Many thanks for your patience

**Comment by solskogen** [comment]
Why aren't you doing what the README.md says?

**Comment by Bazil79** [comment]
Dont understand! Read me dependencies says to install the framework which I downloaded and unzipped. Ran the install for it. Done the terminal command but got an error. What in the readme am I missing or am I supposed to run all the command lines listed.

I'm getting frustrated with myself on this because I'm not very good at this sort of thing and wanted to try something new

Thanks again for your advice and patience
